src/components/QueryNot/index.js

```diff
@@ -2,6 +2,10 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Query from '../Query';
 
+/**
+* The QueryNot predicate checks that the path doesn't match the provided value exactly.
+* It takes a single value as the argument.
```

Comment:
`It takes a single value as the argument`
Again same a react component does not take an argument it take props and this specifically is explaining the prop value which is already explained.